src/ArrayValueCalculator.java

```diff
@@ -0,0 +1,28 @@
+public class ArrayValueCalculator {
+
+    public static int doCalc(String[][] matrix) {
+            if (matrix == null) {
+                throw new NullPointerException("The array must not be null, the array must be 4x4");
```

Comment:
The message is a bit strange because here you're just checking whether `matrix` is null or not. So, having it like this - `The array must not be null!` is more than enough.